roles/common/defaults/main.yaml

```diff
@@ -1,21 +1,25 @@
 ---
 common_debug_massage: "hElLo tHeRe"
 
-# NTP Role
-ntp_package: ntp
-ntp_service: ntpd
-ntp_configfile: /etc/ntp.conf
-ntp_configfile_user: root
-ntp_configfile_group: root
-ntp_timezone: Europe/Kiev
+# Packages that would be installed
+package_list:
```

Comment:
You can leave it like that. But if there is only one element, then I don’t see why it is necessary to add complexity in the form of a loop.
In addition, it is recommended to add the name of the role to the beginning of each variable, e.g. `common_ntp_timezone`

Reply:
fixed (variables)
and pkg without loop